sklearn/ensemble/tests/test_forest.py

```diff
@@ -197,7 +196,11 @@ def test_probability():
         yield check_probability, name
 
 
-def check_importances(name, criterion, X, y):
+def check_importances(name, criterion, X, y, dtype):
```

Comment:
Sorry I don't follow you. I thought since there were `check_*` functions, the parameters of the functions show up in pytest and it's easier to debug? Which is why I included them as parameters rather than looping them.